### ISSUE TITLE
Add documentation about how to recover a trapped token using the CLI

### ISSUE THREAD

**Issue by john-light** [aragon/aragon-wiki#439]
It is currently not possible from the Aragon GUI to recover tokens that are trapped e.g. sent directly to the Finance app address instead of deposited correctly through the app interface.

We should add a section to the Troubleshooting section of the User Guide that will tell users how to recover these tokens using the CLI, since that is the only way to do it right now. Alternatively, we can add a section to the CLI documentation directly about how to do this.

cc @0xGabi

**Comment by sohkai** [comment]
For context, each app exposes an [`transferToVault()`](https://github.com/aragon/aragonOS/blob/dev/contracts/common/VaultRecoverable.sol#L26) function that allows them to do this, so it should be fairly simple to do with `dao exec`.

**Comment by 0xGabi** [comment]
You are right @john-light this is a recurring subject so make sense to add it in the Troubleshooting guide. 

>For context, each app exposes an transferToVault() function that allows them to do this, so it should be fairly simple to do with dao exec.

@sohkai You are also right. What is currently happening that is adding a layer of complexity to the hole process is that the ABI of the finance app are not propagate into ipfs so people need to pin the `artifact.json` manually or use the new command `ipfs propagate` to been able to use `dao exec` (the same happened to melon team with their `agent app`). I think this is a good subject to further explain how ipfs works with aragon and introduce in practice the new commands: `apm info`, `apm packages`, `ipfs view` and `ipfs propagate`.

**Comment by sohkai** [comment]
We could also provide a baseline "AragonApp" abi bundled with the CLI, so people can invoke at least those if the ipfs fetch fails.

**Comment by john-light** [comment]
Thanks y'all for the input on this. I am going to leave this issue and once [#122](https://github.com/aragon/hack/issues/122) that Gabi opened is closed I will add a link to that documentation to the troubleshooting section of the user guide.
